README.md

```diff
@@ -366,28 +357,33 @@ aws sqs --no-sign-request --region local --endpoint-url http://localhost:4566/00
 Open Temporal UI in a browser by entering the
 URL: http://localhost:8088/namespaces/chainstorage-ethereum-mainnet/workflows
 
+Connect to the admin docker container:
+```shell
+docker exec -it chainstorage-_api_1 /bin/sh
```

Comment:
1. Why access to  `api` here, shouldn't be `admin`?
2. This `chainstorage-_api_1` format doesn't look correct to me, could you double check?

Reply:
Good catch